### ISSUE TITLE
Matlab testing of file-encoding input

### ISSUE THREAD

**Issue by cbm755** [catch22/octave-doctest#259]
re: Matlab: have not thought at all about encoding on Matlab, but its a good point.  My test (`test_bytecount_CP1252.m`) is almost certainly Octave specific, as tests for utf-8 usage internally.

I don't currently have an easy way to test this on Matlab: suggest we worry about Octave for now, then happy to revert to your inline testing style of `run_tests.m` as you originally had it later...

_Originally posted by @cbm755 in https://github.com/catch22/octave-doctest/issues/258#issuecomment-1187846283_

**Comment by cbm755** [comment]
Here's the code I took out in commits:

9e578b31dd6791aab94fad632fe5ab3ecda6abd7

716c392fe1c1c58d96f4177d346cf788db0c5854

```octave

success_extra = true;

%% test for file that is not encoded in UTF-8
% A bug in Octave 7 requires that the folder containing the .oct-config file
% is in the load path (not the current folder).
path_orig = path ();
path_protect = onCleanup (@() path (path_orig));

addpath (canonicalize_file_name ("test_encoding"));
success_extra = doctest ("test_CP1252.m") && success_extra;

clear path_protect;
```
Perhaps we want to put that back into the body of `run_tests.m`, try it on Matlab.  Then the bists will test on Octave..

**Comment by cbm755** [comment]
Depends on #213

**Comment by cbm755** [comment]
Fixed in #273.
